scripts/test-runner.js

```diff
@@ -103,21 +104,49 @@ function execExternal(externalSpec, code) {
   return String(output.trim());
 }
 
+function argumentCodeWithLazyObjectSupport(code) {
```

Comment:
Name not clear, did you mean "augment"?

Reply:
Thank you! That's the word I am looking for.